client/components/deposits-overview/next-deposit.tsx

```diff
@@ -31,6 +30,88 @@ type NextDepositProps = {
 	overview?: AccountOverview.Overview;
 };
 
+const DepositIncludesLoanPayoutNotice = () => (
+	<BannerNotice
+		status="warning"
+		icon={ <InfoOutlineIcon /> }
+		isDismissible={ false }
+	>
+		{ interpolateComponents( {
+			mixedString: __(
+				'This deposit will include funds from your WooCommerce Capital loan. {{learnMoreLink}}Learn more{{/learnMoreLink}}',
+				'woocommerce-payments'
+			),
+			components: {
+				learnMoreLink: (
+					// Link content is in the format string above. Consider disabling jsx-a11y/anchor-has-content.
+					// eslint-disable-next-line jsx-a11y/anchor-has-content
+					<a
+						href={
+							'https://woocommerce.com/document/woocommerce-payments/stripe-capital/overview'
+						}
+						target="_blank"
+						rel="noreferrer"
+					/>
+				),
+			},
+		} ) }
+	</BannerNotice>
+);
+
+const NewAccountWaitingPeriodNotice = () => (
+	<BannerNotice
+		status="warning"
+		icon={ <NoticeOutlineIcon /> }
+		className="new-account-waiting-period-notice"
+		isDismissible={ false }
+	>
+		{ interpolateComponents( {
+			mixedString: __(
+				'Your first deposit is held for seven business days. {{whyLink}}Why?{{/whyLink}}',
+				'woocommerce-payments'
+			),
+			components: {
+				whyLink: (
+					// Link content is in the format string above. Consider disabling jsx-a11y/anchor-has-content.
+					// eslint-disable-next-line jsx-a11y/anchor-has-content
+					<a
+						target="_blank"
+						rel="noopener noreferrer"
+						href="https://woocommerce.com/document/woocommerce-payments/deposits/deposit-schedule/#section-1"
+					/>
+				),
+			},
+		} ) }
+	</BannerNotice>
+);
+
+const NegativeBalanceDepositsPausedNotice = () => (
+	<BannerNotice
+		status="warning"
+		icon={ <NoticeOutlineIcon /> }
+		className="negative-balance-deposits-paused-notice"
+		isDismissible={ false }
+	>
+		{ interpolateComponents( {
+			mixedString: __(
+				'Deposits may be interrupted while your WooCommerce Payments balance remains negative. {{whyLink}}Why?{{/whyLink}}',
+				'woocommerce-payments'
+			),
+			components: {
+				whyLink: (
+					// Link content is in the format string above. Consider disabling jsx-a11y/anchor-has-content.
+					// eslint-disable-next-line jsx-a11y/anchor-has-content
+					<a
+						target="_blank"
+						rel="noopener noreferrer"
+						href="https://woocommerce.com/document/woocommerce-payments/fees-and-debits/account-showing-negative-balance/"
+					/>
+				),
+			},
+		} ) }
+	</BannerNotice>
+);
+
```

Comment:
I factored these out to keep the main component slim, reduce indentation, and make the show/hide logic easy to read.